lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +250,75 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        self.main_text_bs = article_soup.find('div', {
+            'itemprop': 'articleBody'}).text.replace('\n\n', ' ').replace('  ', '').strip()
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        self.article.title = article_soup.find('div', {'class', 'b-news-detail-top'}).find('h1').text
+
+        author_info = article_soup.find('div', {'class': "b-meta-item b-meta-item--bold"}).find('span',
+                                                                                                {'itemprop': 'name'})
+        if author_info:
+            self.article.author = author_info.text.strip()
+
+        date_info = article_soup.find('time', {'class': "b-meta-item"}).text.strip()
```

Comment:
instead of .text.strip() you coud use `get_text(strip=True)`

Reply:
okay